PlaygroundService/Grains/PlaygroundGrain.cs

```diff
@@ -18,6 +20,100 @@ public class PlaygroundGrain : Grain, IPlaygroundGrain
     {
         _logger = logger;
     }
+    
+    public async Task <string> GenerateZip(string template, string templateName)
+    {
+        var tempPath = Path.GetTempPath();
+        var templatePath = Path.Combine(tempPath, Path.GetFileNameWithoutExtension(template), Guid.NewGuid().ToString());
+        var sourceFolder = templatePath + "/src";
+        var command = "dotnet new --output " + templatePath + " " + template + " -n " + templateName;
+        try
+        {
+            var startInfo = new ProcessStartInfo
+            {
+                FileName = "/bin/bash", // We use bash to execute commands
+                Arguments = $"-c \"{command}\"", // -Option c allows bash to execute a string command
+                UseShellExecute = false,
+                RedirectStandardOutput = true, // If necessary, you can redirect the output to the C # program
+                CreateNoWindow = true // Do not create a new window
+            };
+
+            // Using the Process class to start a process
+            using (var process = new Process { StartInfo = startInfo })
+            {
+                process.Start(); // start process
+                // If necessary,can read the output of the process
+                // string output = process.StandardOutput.ReadToEnd();
+                process.WaitForExit(); // Waiting for process to exit
+            }
+            _logger.LogInformation("PlayGroundGrain GenerateZip  dotnet new end command: " + command + " time: " + DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss"));
+        
+            ZipFile.CreateFromDirectory(sourceFolder, templatePath + "/src.zip");
```

Comment:
have to zip the whole project, i.e. src and test folders.

Reply:
done